<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0" tests="4" failures="4">
  <testsuite name="Root Suite" timestamp="2020-03-22T20:50:20" tests="0" file="cypress/integration/apps/projects/PR19.js" failures="0" time="0">
  </testsuite>
  <testsuite name="19 - I can edit unsubmiteted experiments" timestamp="2020-03-22T20:50:20" tests="4" failures="4" time="0">
    <testcase name="19 - I can edit unsubmiteted experiments PR-19-1 Should edit name" time="0" classname="PR-19-1 Should edit name">
      <failure message="Timed out retrying: Expected to find content: &apos;PR19 name&apos; within the element: &lt;div.section&gt; and with the selector: &apos;.row&apos; but never did." type="CypressError"><![CDATA[CypressError: Timed out retrying: Expected to find content: 'PR19 name' within the element: <div.section> and with the selector: '.row' but never did.
    at cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138017:3)
    at retry (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:132299:19)
    at onFailFn (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:120514:16)
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:163008:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
    <testcase name="19 - I can edit unsubmiteted experiments PR-19-2 Should edit description" time="0" classname="PR-19-2 Should edit description">
      <failure message="Timed out retrying: Expected to find content: &apos;PR19 edit&apos; within the element: &lt;div.section&gt; and with the selector: &apos;.row&apos; but never did." type="CypressError"><![CDATA[CypressError: Timed out retrying: Expected to find content: 'PR19 edit' within the element: <div.section> and with the selector: '.row' but never did.
    at cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138017:3)
    at retry (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:132299:19)
    at onFailFn (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:120514:16)
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:163008:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
    <testcase name="19 - I can edit unsubmiteted experiments PR-19-3 Should add tags" time="0" classname="PR-19-3 Should add tags">
      <failure message="Timed out retrying: Expected to find content: &apos;PR19 edit&apos; within the element: &lt;div.section&gt; and with the selector: &apos;.row&apos; but never did." type="CypressError"><![CDATA[CypressError: Timed out retrying: Expected to find content: 'PR19 edit' within the element: <div.section> and with the selector: '.row' but never did.
    at cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138017:3)
    at retry (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:132299:19)
    at onFailFn (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:120514:16)
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:163008:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
    <testcase name="19 - I can edit unsubmiteted experiments PR-19-4 Should delete tags" time="0" classname="PR-19-4 Should delete tags">
      <failure message="Timed out retrying: Expected to find content: &apos;PR19 edit&apos; within the element: &lt;div.section&gt; and with the selector: &apos;.row&apos; but never did." type="CypressError"><![CDATA[CypressError: Timed out retrying: Expected to find content: 'PR19 edit' within the element: <div.section> and with the selector: '.row' but never did.
    at cypressErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138036:9)
    at throwErr (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:137969:11)
    at Object.throwErrByPath (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:138017:3)
    at retry (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:132299:19)
    at onFailFn (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:120514:16)
    at tryCatcher (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:164895:23)
    at Promise._settlePromiseFromHandler (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162831:31)
    at Promise._settlePromise (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162888:18)
    at Promise._settlePromise0 (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:162933:10)
    at Promise._settlePromises (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:163008:18)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueue (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159620:16)
    at Async../node_modules/bluebird/js/release/async.js.Async._drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159630:10)
    at Async.drainQueues (http://127.0.0.1:8080/__cypress/runner/cypress_runner.js:159504:14)]]></failure>
    </testcase>
  </testsuite>
</testsuites>